app/models/service_awx.rb

```diff
@@ -0,0 +1,84 @@
+class ServiceAwx < Service
```

Comment:
Similarly can we `ServiceAnsibleTower < ServiceAwx` here?  (would require ActsAsStiLeafClass in this case)